packages/uploadthing/src/mime-types/index.ts

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-var-requires */
+
+/**
+ * Vendored version of mime-types that can run on the edge due to not using path.extname
+ *
+ * Also ported it to TypeScript cause it was easier than playing around with custom d.ts file
+ *
+ * Removed all the stuff we didn't use
+ */
+
+/*!
+ * mime-types
+ * Copyright(c) 2014 Jonathan Ong
+ * Copyright(c) 2015 Douglas Christopher Wilson
+ * MIT Licensed
+ */
+
+/**
+ * Module dependencies.
+ * @private
+ */
+
+const db = require("./db.json");
+function extname(path: string) {
+  const index = path.lastIndexOf(".");
+  return index < 0 ? "" : path.substring(index);
+}
```

Comment:
this was `path.extname` before - you decide if we need a more sophisticated version of this